Bubbkesort_week6/spec/spec_helper.rb

```diff
@@ -0,0 +1,98 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
remove all commented line
![Uploading Screenshot 2022-04-20 at 08.40.17.png…]()

Reply:
fixed it